docker/pulumi/Dockerfile

```
@@ -166,14 +184,21 @@ ENV GORELEASER_VERSION 1.11.4
 
 SHELL ["/bin/bash", "-o", "errexit", "-o", "nounset", "-o", "pipefail", "-c"]
 
-RUN curl \
+RUN \
+  # Setup environment variables for architecture-specific packages
+  if [ "$TARGETARCH" = "amd64" ]; then \
+   GORELEASER_ARCH=x86_64; \
```

```suggestion
    GORELEASER_ARCH=x86_64; \
```

very nitpicky here, but I think this is missing a space.